--- a/Tests/mocks/devicesettings.h	2026-04-21 19:00:22.239951643 +0000
+++ b/Tests/mocks/devicesettings.h	2026-04-21 19:02:02.376352344 +0000
@@ -72,6 +72,16 @@
     dsAUDIOPORT_TYPE_MAX /**< Maximum index for audio port type. */
 } dsAudioPortType_t;
 
+typedef enum _dsAudioDuckingAction_t {
+    dsAUDIO_DUCKINGACTION_START = 0,
+    dsAUDIO_DUCKINGACTION_STOP  = 1
+} dsAudioDuckingAction_t;
+
+typedef enum _dsAudioDuckingType_t {
+    dsAUDIO_DUCKINGTYPE_ABSOLUTE = 0,
+    dsAUDIO_DUCKINGTYPE_RELATIVE = 1
+} dsAudioDuckingType_t;
+
 typedef enum _dsVideoPortType_t {
     dsVIDEOPORT_TYPE_RF = 0,      ///< RF modulator (channel 3/4) video output          
     dsVIDEOPORT_TYPE_BB,          ///< Baseband (composite, RCA) video output            
@@ -934,6 +944,11 @@
     void enableARC(dsAudioARCTypes_t type, bool enable);
     uint32_t getDolbyVolumeMode() const;
     void setStereoMode(const std::string &mode, bool persist);
+    void setAudioDucking(dsAudioDuckingAction_t action, dsAudioDuckingType_t type, float level) {
+        (void)action;
+        (void)type;
+        (void)level;
+    }
 
 
 
@@ -1444,6 +1459,7 @@
 class Host {
 protected:
     static HostImpl* impl;
+    inline static bool IsInitialized = false;
 
 public:
     static void setImpl(HostImpl* newImpl);
